recipe/patches/0002-clang-add-conda-specific-env-var-CONDA_BUILD_SYSROOT.patch

```diff
@@ -1,12 +1,18 @@
-From c17bbdb9087baecccf7e06e55ac380ae71ee280c Mon Sep 17 00:00:00 2001
+From c0e39291617ecfeef712f51dd19ab986ef85ef94 Mon Sep 17 00:00:00 2001
 From: Nehal J Wani <nehaljw.kkd1@gmail.com>
 Date: Sat, 25 Aug 2018 09:20:04 -0500
 Subject: [PATCH 2/9] clang: add conda specific env var CONDA_BUILD_SYSROOT
 
+And also improve logic for finding the macos sysroot for conda
+
+Co-authored-by: Chris Burr <christopher.burr@cern.ch>
+Co-authored-by: Vincenzo Eduardo Padulano <v.e.padulano@gmail.com>
+Co-authored-by: Fernando Isuru <isuruf@gmail.com>
```

Comment:
It's the other way around.

Reply:
Right, my bad, apologies for that!

Reply:
Please fix it (also in the commit message of https://github.com/conda-forge/clangdev-feedstock/pull/298/commits/a3dcec494e0d298e1d3baee7f01aa7bb495e828c)